### ISSUE TITLE
Wheels are uploaded to TestPyPI instead of normal PyPI

### ISSUE THREAD

**Issue by jackvreeken** [pymola/pymola#30]
Note that we currently upload to the testpypi server (see comment in #28 ). By removing the server: https://test.pypi.org/legacy/ line it will upload to normal PyPI, but the user pymola.wheelbuilder does not have maintainer access there.

@jgoppert
1. Can you give pymola.wheelbuilder maintainer access on PyPI to Pymola?
2. If not, can you change the user+password to your account details?

**Comment by jgoppert** [comment]
It says pymola.wheelbuilder  user doesn't exist.

**Comment by jackvreeken** [comment]
I just tried adding it as a maintainer to the "rtctools" package (which is just an alias), and that works. I just clicked on the package name (to go to its editing page) -> roles -> filled in `pymola.wheelbuilder` as user name, and added it as as a maintainer. If you go to the [rtctools pypi page](https://pypi.python.org/pypi/rtctools/0.0.1.dev2), you see `pymola.wheelbuilder` listed properly as well.
